### PR TITLE
Release v0.1.0 checklist

### PR DESCRIPTION
**Pull request by tridelat**
Todo list before tagging and releasing v0.1.0:
- [x] Fix tests
- [x] Update README.md
- [x] Clean and format CMakeLists.txt
- [x] Fix ROSCO lib path issue
- [x] Check compilation on Linux and Windows

### CODE REVIEW

File: src/elasto/blade_elasto.cpp [hunk 1]
```diff
@@ -220,7 +227,7 @@ void BladeElasto::evaluate_position_rotation(chrono::ChVector<double>& position,
                                              chrono::ChQuaternion<double>& rotation,
                                              int element_index,
                                              double eta) const {
-    auto& element = std::dynamic_pointer_cast<chrono::fea::ChElementBeamTaperedTimoshenko>(elements[element_index]);
+    auto element = std::dynamic_pointer_cast<chrono::fea::ChElementBeamTaperedTimoshenko>(elements[element_index]);
```
Comment (wenchaoyu) [inline src/elasto/blade_elasto.cpp:230]: Just make a remark here;
On Windows we have: 
auto& element = std::dynamic_pointer_cast<chrono::fea::ChElementBeamTaperedTimoshenko>(elements[element_index]);
On Linux we remove "&" to compile it properly.
auto element = std::dynamic_pointer_cast<chrono::fea::ChElementBeamTaperedTimoshenko>(elements[element_index]);

File: src/main.cpp [hunk 2]
```diff
@@ -173,17 +174,17 @@ int main(int argc, char* argv[]) {
...
     seahowl_system.init(system.GetChTime(), dt);
 
-    output_results(seahowl_system, system);
+    int step = 0;
+    output_results(seahowl_system, system, step);
 #ifdef HAVE_VTK
     if (output_vtk) {
         for (auto const& vtk_output : vtk_outputs) {
-            vtk_output.write(time, step);
+            vtk_output.write(system.GetChTime(), step);
```
Comment (wenchaoyu) [inline src/main.cpp:182]: It seems to have an issue in write function of vtk_output:
  - only blade 3 data is written, blade 1 and 2 are not saved in vtk folder
  - lots of files "_xxxx.vtu" are written outside of vtk folder

**Review by wenchaoyu — CHANGES_REQUESTED** [review]

File: src/main.cpp (hunk 2 above)
Comment (tridelat) [reply src/main.cpp:182]: Fixed on [02592e1](https://github.com/Total-RD/seahowl/pull/32/commits/02592e1d0c60f16e61c89ce0f70d351a5a789f40)

**Review by wenchaoyu — APPROVED** [review]
It is ok for me, thanks!

**Pull request merged**